### PR TITLE
Clock

### PR DESCRIPTION
**Pull request by keawade**
left-pad dependency removed. :tada:

Resolves #14. See #8.

### CODE REVIEW

**Comment by keawade** [comment]
Finally got back to it. @NathanBland, I think this resolves your concerns?

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-spec-reporter": "0.0.24",
     "karma-webpack": "^1.7.0",
+    "left-pad": "^1.1.1",
```
Comment (NathanBland) [inline package.json]: this is still here :wink:

Comment (keawade) [inline package.json]: That's what I get for trusting npm to remove it for me.

**Comment by NathanBland** [comment]
That's more like it! :+1: 

I still don't get why you didn't use `toLocaleTimeString`, but whatever floats your boat. I'll merge this in.

**Comment by keawade** [comment]
What I had worked and I was approximately two weeks behind on getting this in haha.

**Pull request merged**